lib/ui_kit/organisms/cards/ui_kit_find_someone_card.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:shuffle_uikit/shuffle_uikit.dart';
+
+class UiKitFindSomeoneCard extends StatelessWidget {
+  final String? avatarUrl;
+  final String? userNickName;
+  final String? userName;
+  final int? userPoints;
+  final int? sameInterests;
+  final VoidCallback? onMessage;
+
+  const UiKitFindSomeoneCard({
+    super.key,
+    this.avatarUrl,
+    this.userNickName,
+    this.userName,
+    this.userPoints,
+    this.sameInterests,
+    this.onMessage,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    double calculatedWidth = MediaQuery.of(context).size.width;
+    final boldTextTheme = context.uiKitTheme?.boldTextTheme;
+
+    return Material(
+      color: ColorsFoundation.surface3,
+      borderRadius: BorderRadiusFoundation.all24,
+      clipBehavior: Clip.hardEdge,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Expanded(
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    CircularAvatar(
+                      avatarUrl: avatarUrl ?? '',
+                      height: calculatedWidth * 0.2,
+                    ),
+                    Text(
+                      '@$userNickName',
+                      style: boldTextTheme?.caption2Bold,
+                    ),
+                    SpacingFoundation.verticalSpace4,
+                    Text(
+                      userName ?? '',
+                      style: boldTextTheme?.caption2Bold
+                          .copyWith(color: Colors.grey),
```

Comment:
that's no a grey - check ColorsFoundation or UiKitColors and find matching color there (from figma take the code)